<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MasternodeManager</class>
 <widget class="QWidget" name="MasternodeManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>723</width>
    <height>457</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="topLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>Masternodes Network</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout">
         <item row="1" column="0">
          <widget class="QTableWidget" name="tableWidget">
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
           <property name="selectionBehavior">
            <enum>QAbstractItemView::SelectRows</enum>
           </property>
           <attribute name="horizontalHeaderStretchLastSection">
            <bool>true</bool>
           </attribute>
           <column>
            <property name="text">
             <string>Address</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Rank</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Active</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Active (secs)</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Last Seen</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Pubkey</string>
            </property>
           </column>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QTableWidget" name="tableWidgetPMN">
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
           <property name="selectionBehavior">
            <enum>QAbstractItemView::SelectRows</enum>
           </property>
           <property name="sortingEnabled">
            <bool>true</bool>
           </property>
           <attribute name="horizontalHeaderStretchLastSection">
            <bool>true</bool>
           </attribute>
           <column>
            <property name="text">
             <string>Address</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Rank</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Active</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Active (secs)</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Last Seen</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Pubkey</string>
            </property>
           </column>
          </widget>
         </item>
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Masternodes Activated:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="countLabel">
             <property name="text">
              <string>0</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="labelPMN">
           <property name="text">
            <string>Parallel Masternodes</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>Masternode Configuration</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="1" column="0">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout"/>
           </item>
           <item>
            <widget class="QTableWidget" name="tableWidget_2">
             <property name="editTriggers">
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
             <property name="alternatingRowColors">
              <bool>true</bool>
             </property>
             <property name="selectionMode">
              <enum>QAbstractItemView::SingleSelection</enum>
             </property>
             <property name="selectionBehavior">
              <enum>QAbstractItemView::SelectRows</enum>
             </property>
             <attribute name="horizontalHeaderStretchLastSection">
              <bool>true</bool>
             </attribute>
             <column>
              <property name="text">
               <string>Alias</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>IP/Onion</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Status</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Collateral Address</string>
              </property>
             </column>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <property name="bottomMargin">
              <number>0</number>
             </property>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QPushButton" name="createButton">
               <property name="text">
                <string>&amp;Create...</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="editButton">
               <property name="text">
                <string>&amp;Edit</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="getConfigButton">
               <property name="text">
                <string>&amp;Get Config</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="removeButton">
               <property name="text">
                <string>&amp;Remove</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QPushButton" name="startButton">
               <property name="text">
                <string>&amp;Start</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="startAllButton">
               <property name="text">
                <string>&amp;Start All</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="stopButton">
               <property name="text">
                <string>&amp;Stop</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="stopAllButton">
               <property name="text">
                <string>&amp;Stop All</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
